.travis.yml

```diff
@@ -10,26 +10,40 @@ branches:
 language: node_js
 
 node_js:
-  - '10'
-  - '12'
+  - 10
+  - 12
 
 install:
   - yarn install --frozen-lockfile --non-interactive
 
 jobs:
   include:
-    - stage: Check TypeScript
+    - &checkTs
```

Comment:
YAML aliases and anchors 🎉 This ensures we actually expand our build matrix to run all stages in both Node 10 and Node 12.